ts/client/features/base.ts

```diff
@@ -31,10 +31,10 @@ export abstract class BaseCommitment implements Commitment {
 }
 
 export class ConcreteBatch<T extends Commitment> implements Batch {
-    private tree: Tree;
+    private tree: MemoryTree;
```

Comment:
This can be the `Tree` interface

Reply:
It uses `.merklize` static method from the MemoryTree class. Thought it would be better to give keep the type of tree as MemoryTree due to that method. What do you think?

Reply:
We can make the `merklize` function return the `Tree` interface and still use that here. Fine as is, can do on next PR.